platforms/Windows/bld/bld.wixproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="WixToolset.Sdk/4.0.1">
+  <PropertyGroup>
+    <DefineConstants>
+      $(DefineConstants);
+      DEVTOOLS_ROOT=$(DEVTOOLS_ROOT);
+      TOOLCHAIN_ROOT=$(TOOLCHAIN_ROOT);
+      TOOLCHAIN_ROOT_USR_LIB_CLANG=$(TOOLCHAIN_ROOT_USR_LIB_CLANG);
```

Comment:
Not sure if there is a better way to do this, IIRC this was required for the use of heat.